docs/docsify/sidebar.md

```diff
@@ -1,6 +1,7 @@
 <!-- Sidebar for Docsify -->
 
 * [**Home**](/ "Node.js Undici")
+* [Examples](/examples/ "Undici Examples")
```

Comment:
This looks like this

![image](https://github.com/nodejs/undici/assets/5059100/7bdb0977-26c2-4b79-b379-e3268c4bce49)

Should we put the examples maybe into best practise?